docs/usage.md

```diff
@@ -8,7 +8,7 @@ The example below allows compiling a Go app and uses [Minify][minify] to concat
 and minify multiple CSS files into a single one.
 
 ```yaml
-version: '3'
+version: "3"
```

Comment:
This was changed this the whole file, but single quotes are really preferred according to out [styleguide](https://taskfile.dev/#/styleguide).

Reply:
Fixed!